# tratando erros

throw


try...catch

finally
instrução sendo chamada independemente do erro